akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -105,6 +105,12 @@ static void lastOptionExample() {
     // #lastOption-operator-example
   }
 
+  static void foldExample() {
+    // #fold
+    Source.range(1, 100).runWith(Sink.fold(0, (res, number) -> res + number), system);
```

Comment:
I'd go with `acc` (as in accumlated) or `sum` and `element` so that it is more clear what parameter is what.